infra/build_specified_commit.py

```diff
@@ -22,75 +22,81 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
 class DockerExecutionError(Exception):
   """An error that occurs when running a docker command."""
 
 
-def build_fuzzer_from_commit(project_name,
-                             commit,
-                             local_store_path,
-                             engine='libfuzzer',
-                             sanitizer='address',
-                             architecture='x86_64',
-                             old_repo_manager=None):
+def build_fuzzers_from_commit(project_name,
+                              commit,
+                              build_repo_manager,
+                              engine='libfuzzer',
+                              sanitizer='address',
+                              architecture='x86_64'):
   """Builds a OSS-Fuzz fuzzer at a  specific commit SHA.
 
   Args:
     project_name: The OSS-Fuzz project name
     commit: The commit SHA to build the fuzzers at
-    local_store_path: The full file path of a place where a temp git repo is stored
+    build_repo_manager: The OSS-Fuzz project's repo manager to be built at
```

Comment:
In a new CL tomorrow. Let's put a period at the end of of every `arg` in a docstring.

Reply:
Sounds good.